src/game/Object/Player.cpp

```diff
@@ -15436,15 +15439,15 @@ void Player::_LoadIntoDataField(const char* data, uint32 startOffset, uint32 cou
 
 bool Player::LoadFromDB(ObjectGuid guid, SqlQueryHolder* holder)
 {
-    //       0     1        2     3     4      5       6      7   8      9            10            11
-    // SELECT guid, account, name, race, class, gender, level, xp, money, playerBytes, playerBytes2, playerFlags,"
-    // 12          13          14          15   16           17        18         19         20         21          22           23                 24
+    //       0     1        2     3     4      5       6      7   8      9            10            11				12
+    // SELECT guid, account, name, race, class, gender, level, xp, money, playerBytes, playerBytes2, playerFlags, securitygroup,"
```

Comment:
Why this particular positioning? Why not at the end, instead of changing all the following indexes?

Reply:
I initially wanted to add it at the end, but I wanted it to be obvious when looking at the table with a DBA tool.